sql/core/src/test/scala/org/apache/spark/sql/execution/python/BatchEvalPythonExecSuite.scala

```diff
@@ -23,21 +23,27 @@ import scala.collection.mutable.ArrayBuffer
 import org.apache.spark.api.python.{PythonEvalType, PythonFunction}
 import org.apache.spark.sql.catalyst.FunctionIdentifier
 import org.apache.spark.sql.catalyst.expressions.{And, AttributeReference, GreaterThan, In}
-import org.apache.spark.sql.execution.{FilterExec, InputAdapter, SparkPlanTest, WholeStageCodegenExec}
+import org.apache.spark.sql.execution._
+import org.apache.spark.sql.functions.col
 import org.apache.spark.sql.test.SharedSQLContext
 import org.apache.spark.sql.types.BooleanType
 
 class BatchEvalPythonExecSuite extends SparkPlanTest with SharedSQLContext {
```

Comment:
I don't think your tests should be in this suite since it is just for `BatchEvalPythonExec`.  How about `ExtractPythonUDFsSuite`?